### PR TITLE
Release/extensions

### PR DESCRIPTION
**Pull request by RamyaR29**
This PR fixes #

### CODE REVIEW

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -4521,33 +4521,33 @@
 					"versions": [
 						{
 							"version": "3.0.8",
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json:4523]: Update this

Comment (RamyaR29) [reply extensionsGallery-insider.json]: updated, thanks

**Comment by RamyaR29** [comment]
> @microsoft-github-policy-service agree company="Microsoft"

**Comment by Charles-Gagnon** [comment]
@RamyaR29 You have to remove the `>` from your response

**Pull request merged**